--- lib7zip-3.0.0/CMakeLists.txt.orig	2018-09-04 07:35:37.000000000 +0200
+++ lib7zip-3.0.0/CMakeLists.txt	2018-12-31 13:09:02.058335061 +0100
@@ -39,7 +39,7 @@
 
 
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-  set(warnings "-Wall -Wextra -Werror -Wno-unused-parameter -Wno-class-memaccess")
+  set(warnings "-Wall -Wextra -Werror -Wno-unused-parameter")
   set(no_undefined "-Wl,--no-undefined")
 elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
   set(warnings "-Wall -Wextra -Werror -Wno-inconsistent-missing-override -Wno-unused-parameter")
